### PR TITLE
Fix uninstall playbook

### PR DESCRIPTION
**Pull request by mkumatag**
Fixes - https://github.com/kubernetes/contrib/issues/1674

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/contrib/1694)

### CODE REVIEW

**Comment by mkumatag** [comment]
@ingvagabund

File: ansible/playbooks/adhoc/uninstall.yml [hunk 1]
```diff
@@ -114,7 +114,7 @@
     - name: Remove remaining files
       file: path={{ item }} state=absent
       with_items:
-        - "~{{ ansible_ssh_user }}/.kube"
+        - "{{ ansible_env.HOME }}/.kube"
```
Comment (ingvagabund) [inline ansible/playbooks/adhoc/uninstall.yml]: What is the difference between ``~/.kube`` and ``{{ ansible_env.HOME }}/.kube`` then?

Comment (mkumatag) [inline ansible/playbooks/adhoc/uninstall.yml]: I feel both are same. :)

Comment (ingvagabund) [inline ansible/playbooks/adhoc/uninstall.yml]: Got the same feeling :). Reading through stackoverflow articles and ansible docs and can't find anything particular and usable for the ``ansible_env.HOME`` case. So far have not found anything saying it is incorrect usage. With the ``ansible_env.HOME`` it is explicitly mentioned I want remote host directory which corresponds to a user under which the ansible is ssh'ed.

**Comment by ingvagabund** [comment]
Looks good, thanks.

**Pull request merged**